### PR TITLE
Add an example to run multi-worker training with distribution strategies.

### PR DESCRIPTION
**Pull request by yuefengz**

### CODE REVIEW

File: distribution_strategy/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+This directory provides an example of running multi-worker training with
+Distribution Strategies.
+
+Please first read the
+[documentation](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/distribute/README.md#multi-worker-training)
+of Distribution Strategy for multi-worker training. We also assume that readers
+of this page have background of [Google Cloud](https://cloud.google.com/) and
+its [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/).
+
+This directory contains the following file:
```
Comment (anj-s) [inline distribution_strategy/README.md]: s/file/files

Comment (yuefengz) [reply distribution_strategy/README.md]: Done.

File: distribution_strategy/tf_std_server.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+def main(unused_argv):
+  to_import_contrib_ops = tf.contrib.resampler
+
+  # Load you custom ops here.
+
```
Comment (anj-s) [inline distribution_strategy/tf_std_server.py]: remove empty line

File: distribution_strategy/tf_std_server.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+"""Run a standard tensorflow server."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+def main(unused_argv):
+  to_import_contrib_ops = tf.contrib.resampler
```
Comment (anj-s) [inline distribution_strategy/tf_std_server.py]: What is this for?

Comment (yuefengz) [reply distribution_strategy/tf_std_server.py]: This is load contrib ops. Added a comment.

File: distribution_strategy/tf_std_server.py (hunk 2 above)
Comment (yuefengz) [reply distribution_strategy/tf_std_server.py]: Added a comment.

**Comment by jhseu** [comment]
@frankchn FYI to see how he's handling Kubernetes -> ClusterSpec

**Pull request merged**